Gets Team names in Ranked

Jxfe opened the pull request:
Calls '/chat/v5/participants/champ-select' to get other players name in lobby and links a u.gg to all profiles.

musicderp commented on src/eel_driver.py:
```diff
@@ -5,12 +5,41 @@
+@eel.expose
+def get_ranked_names():
+    # Find the LeagueClient.exe process and extract the required information
+    for process in psutil.process_iter():
```
So as you can see from the way we write our code, we use specific files for our functions and then call them in order to keep the eel_driver.py file clean. A good place for this function is in a file in the utilities folder that you can import and call

musicderp commented on src/eel_driver.py:
```diff
@@ -5,12 +5,41 @@
+            break
+
+    # Create URL and headers for the request
+    url = f'https://127.0.0.1:{port}/chat/v5/participants/champ-select'
```
We're using willump to establish a connection, you can just do client.request rather than doing this all yourself, again take a look at our functions for how we do this

musicderp commented on src/eel_driver.py:
```diff
@@ -5,12 +5,41 @@
+    response = requests.get(url=url, headers=headers, verify=r"{}\riotgames.pem".format(os.getcwd()))
+
+    # Parse the JSON response and extract the names
+    response_data = json.loads(''.join(chunk.decode() for chunk in response))
```
This conversion ends up not being necessary, you can do response.json once you switch to using willump

musicderp commented on src/requirements.txt:
```diff
@@ -30,3 +30,6 @@ wincertstore==0.2
 yarl==1.7.2
 zope.event==4.5.0
 zope.interface==5.4.0
+os 
```
make sure to use pip freeze > requirements.txt so that it includes the version we are currently using

musicderp commented on src/riotgames.pem:
```diff
@@ -0,0 +1,25 @@
+-----BEGIN CERTIFICATE-----
```
Don't commit this, no point

musicderp commented on src/utils/ui/debug.html:
```diff
@@ -59,6 +59,12 @@
             </a>
             <span class="tooltip">Autopilot</span>
         </li>
+        <li>
```
Let's use Teammate Names rather than Team Names

musicderp commented on src/utils/ui/get_names.js:
```diff
@@ -0,0 +1,44 @@
+$(document).ready(function () {
```
You should try to use native jquery syntax here, while this works, it's not ideal

musicderp commented on src/utils/ui/main.html:
```diff
@@ -59,6 +59,12 @@
             </a>
             <span class="tooltip">Autopilot</span>
         </li>
+        <li>
```
Teammate Names

musicderp reviewed (changes requested):
Good first PR, but doesn't quite match the style we've used

Jxfe commented:
Added file to util and modified the js having issues with the new client.request where itd say Session is closed? am i doing something wrong there in /src/utik/ranked_names.py
